R/colourInput.R

```diff
@@ -26,6 +26,8 @@
 #' @param allowedCols A list of colours that the user can choose from. Only
 #' applicable when \code{palette == "limited"}. The \code{limited} palette
 #' uses a default list of 40 colours if \code{allowedCols} is not defined.
+#' @param allowAlpha If \code{TRUE}, enables a slider to choose a transparency amount
```

Comment:
Thanks for adding doc. Could you also run roxygen so that the actual documentation file gets updated with this?